chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,671 @@
+    "|$e$          |noise                    |1-dimensional  ​|\n",
+    "\n",
+    "This problem is:\n",
+    "* Considered a Linear inverse problem since the forward model is linear.\n",
+    "* Ill-posed (in the sense of Hadamard [1]) since the solution is not unique, i.e., for some given value of $b$, e.g., $b=3$, all points $(x_1, x_2)$ that satisfy $x_1 + x_1 = 3$ are solutions to the (noise-free) problem."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us define the BIP components and solve it using CUQIpy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "hide-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Importing the required libraries\n",
+    "from cuqi.distribution import Gaussian\n",
+    "from cuqi.problem import BayesianProblem\n",
+    "from cuqi.model import LinearModel\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 1.1. The forward model </font> <a name=\"r-the-forward-model\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us define the forward model $\\mathbf{A}$, we first define the matrix:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "A_matrix = np.array([[1.0, 1.0]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we wrap `A_matrix` in a `CUQIpy` forward model object, we name `A`, as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "A = LinearModel(A_matrix)\n",
+    "print(A)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 1.2. The prior </font> <a class=\"anchor\" id=\"r-the-prior\"></a>\n",
+    "\n",
+    "**Bayesian approach: Use prior to express belief about solution**\n",
+    "\n",
+    "A common choice for simplicity is the zero mean Gaussian prior where the components are independent and identically distributed (i.i.d.):\n",
+    "\n",
+    "$$ \\mathbf{x} \\sim \\mathrm{Gaussian}(\\mathbf{0}, \\delta^2 \\mathbf{I})$$\n",
+    "\n",
+    "The probability density function (PDF) of such a Gaussian prior is expressed as\n",
+    "\n",
+    "$$ \\pi (\\mathbf{x}) = \\frac{1}{\\sqrt{(2 \\pi)^n \\delta^{2n}}} \\mathrm{exp}\\left(-\\frac{||\\mathbf{x}||^2}{2\\delta^2}\\right) $$\n",
+    "\n",
+    "where:\n",
+    "- $n$ is the dimension of the parameter space (which is 2 in this specific case),\n",
+    "- $\\delta$ is the standard deviation of the prior distribution,\n",
+    "- $\\mathbf{I}$ is the identity matrix.\n",
+    "\n",
+    "Let us define the prior distribution for the parameters $x_1$ and $x_2$:"
```

Comment:
Just curious if we want to write x1 and x2 explicitly here or just x. Probably make it clear that we are now creating the prior in CUQIpy or with the CUQIpy Gaussian distribution?

Reply:
done :)